packages/turf-shortest-path/package-lock.json

```diff
@@ -0,0 +1,1981 @@
+{
```

Comment:
I've recently just been removing/excluding the `package-lock.json` files, I'm not too sure if we should be having two `lock` files (yarn + npm).

I've added `package-lock.json` to the `.gitignore`, we could always add it back, but it seems like two lock files is a bit overkill (we only need to stick with one).

https://github.com/Turfjs/turf/commit/593cb914494ae24161895db652be4519aeba3907